### ISSUE TITLE
[Testnet stuck block 51896] error calling MarshalJSON: year outside of range [0,9999]

### ISSUE THREAD

**Issue by jun0tpyrc** [stellar/go#3180]
`stellar/quickstart:testing@sha256:c6e13d236b542f8eb43cebcb36b1b1a44a7b888ea35f76386a981d11e211d362`

horizon-stdout
```
time="2020-11-01T00:08:35.686Z" level=info msg="Processing ledger" commit=true ledger=true pid=153 sequence=51896 service=expingest state=true
time="2020-11-01T00:08:35.692Z" level=info msg="Trimmed offers table" offer_rows_removed=0 pid=153 service=expingest
time="2020-11-01T00:08:35.696Z" level=error msg="Error in ingestion state machine" current_state="resume(latestSuccessfullyProcessedLedger=51895)" error="Error running processors on ledger: Error commiting changes from processor: error in *processors.ClaimableBalancesProcessor.Commit: error executing batch: error adding values while inserting to claimable_balances: exec failed: sql: converting argument $3 type: json: error calling MarshalJSON for type xdr.ClaimPredicate: json: error calling MarshalJSON for type *time.Time: Time.MarshalJSON: year outside of range [0,9999]" next_state="resume(latestSuccessfullyProcessedLedger=51895)" pid=153 service=expingest
time="2020-11-01T00:08:36.696Z" level=info msg="Ingestion system state machine transition" current_state="resume(latestSuccessfullyProcessedLedger=51895)" next_state="resume(latestSuccessfullyProcessedLedger=51895)" pid=153 service=expingest
```

**Comment by bartekn** [comment]
This is Horizon issue. Will be fixed in 1.11.0 (tomorrow).

**Comment by sammy1991106** [comment]
Same issue here.

**Comment by tundak** [comment]
Still waiting for fix.

**Comment by leighmcculloch** [comment]
This has been fixed in [v1.11.0](https://github.com/stellar/go/releases/tag/horizon-v1.11.0) released today. It looks like we missed updating this duplicate issue. The fix was implemented in #3183.

I'm closing this issue, please reopen if the issue continues to occur.

**Comment by jun0tpyrc** [comment]
> This has been fixed in [v1.11.0](https://github.com/stellar/go/releases/tag/horizon-v1.11.0) released today. It looks like we missed updating this issue. The fix was implemented in #3183.
> 
> I'm closing this issue, please reopen if the issue continues to occur.


Hello @can we also have a new docker image of `testing` tag with the integration fixed, 

the latest horizon seem changed something asking for DATABASE_URL,
 so just mounting the binary into old image like below would not work
`"/data/stellar/app/horizon-v1.11.0-linux-amd64/horizon:/opt/stellar/horizon/bin/horizon"`

it requires crafting `DATABASE_URL` envvar, after which I still couldn't apply migration automatically/manually
```
starting horizon...
2020/11/03 05:24:51 There are 44 migrations to apply in the "up" direction.
2020/11/03 05:24:51 The necessary migrations are: [1_initial_schema.sql 2_index_participants_by_toid.sql 3_use_sequence_in_history_accounts.sql 4_add_protocol_version.sql 5_create_trades_table.sql 6_create_assets_table.sql 7_modify_trades_table.sql 8_add_aggregators.sql 8_create_asset_stats_table.sql 9_add_header_xdr.sql 10_add_trades_price.sql 11_add_trades_account_index.sql 12_asset_stats_amount_string.sql 13_trade_offer_ids.sql 14_fix_asset_toml_field.sql 15_ledger_failed_txs.sql 16_ingest_failed_transactions.sql 17_transaction_fee_paid.sql 18_account_for_signers.sql 19_offers.sql 20_account_for_signer_index.sql 21_trades_remove_zero_amount_constraints.sql 22_trust_lines.sql 23_exp_asset_stats.sql 24_accounts.sql 25_expingest_rename_columns.sql 26_exp_history_ledgers.sql 27_exp_history_transactions.sql 28_exp_history_operations.sql 29_exp_history_assets.sql 30_exp_history_trades.sql 31_exp_history_effects.sql 32_drop_exp_history_tables.sql 33_remove_unused.sql 34_fee_bump_transactions.sql 35_drop_participant_id.sql 36_deleted_offers.sql 37_add_tx_set_operation_count_to_ledgers.sql 38_add_constraints.sql 39_claimable_balances.sql 39_history_trades_indices.sql 40_fix_inner_tx_max_fee_constraint.sql 41_add_sponsor_to_state_tables.sql 42_add_num_sponsored_and_num_sponsoring_to_accounts.sql]
2020/11/03 05:24:51 A database migration is required to run this version (v1.11.0) of Horizon. Run "horizon db migrate up" to update your DB. Consult the Changelog (https://github.com/stellar/go/blob/master/services/horizon/CHANGELOG.md) for more information.
```

```
root@bstg-xlm-1:/opt/stellar/horizon/bin# ./horizon db migrate up
2020/11/03 05:29:47 pq: relation "offers" already exists handling 19_offers.sql
```

**Comment by tundak** [comment]
`2020/11/03 10:49:15 Invalid config: db-url is blank. Please specify --db-url on the command line or set the DATABASE_URL environment variable.`

after set `DATABASE_URL` as environment variable

`2020/11/03 10:50:24 pq: relation "offers" already exists handling 19_offers.sql`

**Comment by abuiles** [comment]
We just updated the quick starter with the latest version of Horizon. For the people running into the `DATABASE_URL` issue - which version of Horizon did you have previously?

**Comment by bert2002** [comment]
```
Unpacking stellar-horizon (1.11.0-90) over (1.10.1-89) ...
Setting up stellar-horizon (1.11.0-90) ...
2020/11/04 01:30:44 Invalid config: db-url is blank. Please specify --db-url on the command line or set the DATABASE_URL environment variable.
```

**Comment by bartekn** [comment]
Thanks for reports. Tracking in https://github.com/stellar/go/issues/3191.

**Comment by bartekn** [comment]
The fix has been released in Horizon 1.11.1.
